src/Components/Nav/Searchbar/index.js

```diff
@@ -0,0 +1,89 @@
+import React, { Component } from 'react';
+import CategoryView from './CategoryView';
+import ResultView from './ResultView';
+import { categoryData } from '../../../Data/categoryData';
+import { characterData } from '../../../Data/characterData';
+import { fetchGet } from '../../../utils/fetches';
+
+export default class Searchbar extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      serachKeyword: '',
+      searchResult: [],
+    };
+  }
+
+  setSerachKeyword = (e) => {
+    this.setState({
+      serachKeyword: e.target.value,
+    });
+  };
+
+  handleReset = () => {
+    this.setState({
+      serachKeyword: '',
+    });
+  };
+
+  handleChangeInput(event) {
+    const searchKeyword = event.target.value;
+    searchKeyword.length === 0 && this.handleReset();
+    fetchGet('', '').then((res) =>
+      this.setState({
+        searchResult: res.result,
+      }),
+    );
+  }
```

Comment:
```suggestion
  handleChangeInput(event) {
    const searchKeyword = event.target.value;
    
    if (!searchKeyword.length) {
      this.handleReset();
    }
    
    fetchGet('', '').then((res) =>
      this.setState({
        searchResult: res.result,
      }),
    );
  }
```
논리적으로 코드 진행의 분기가 생겨야 하는 경우 삼항연산자나 && 연산자 보다 if 문을 쓰는 것이 개인적으로는 더 좋다고 생각합니다.
- 조건문의 실행 결과로 떨어지는 값을 사용하는 경우가 아님 (this.handleReset()는 undefined를 리턴)
- 지금은 한 줄이지만 추후에 두줄 이상의 로직이 될 수도 있는데, 그 경우 어차피 if문으로 수정해야 함

Reply:
수정하였습니다!